### DISCUSSION TITLE
Reality model format (3sm/3mx?)

### DISCUSSION THREAD

**Discussion by ovaisa** [iTwin/itwinjs-core#2253] [Q&A] [answered]
I wanted to know if iTwin supports 3mx format for reality models.  I heard that only 3sm format is supported, wanted to confirm if that is correct.   If I synchronize a dgn which has a 3mx file attached to it or upload 3mx file to Context Share, can I access and display it in iTwin using iTwin API.

Thanks

**Comment by pmconne** [comment, marked as answer]
Yes. Both 3mx and 3sm support publishing as 3d tiles (sometimes referred to as "web-ready tiles"). iModel.js supports all 3d tile formats and it doesn't care what the original reality model format was.
iModel.js does not support displaying .3mx or .3sm files directly.

**Reply by ovaisa** [reply]
Thanks for confirming, glad to know 3mx format is supported (and that I checked before assuming what I heard is correct!)
